Makefile

```diff
@@ -134,7 +134,7 @@ compressed: $(ALL_COMPRESSED_FILES)
 check_compress_tool:
 ifdef MISSING_ZOPFLI
   ifdef MISSING_OPTIPNG
-	$(error "neither $(ZOPFLIPNG) nor $(OPTIPNG) is available")
+	$(error "neither $(ZOPFLIPNG) nor $(OPTIPNG) is available. For installation instructions, see README.md")
```

Comment:
it's always an error to use optipng; we should just remove that option [perhaps as a separate PR]

Reply:
Ahh I did not know that. For now, shall I make an issue for it?

Reply:
yes plz

Reply:
#302